### DISCUSSION TITLE
Why am I only getting first 6 posts of this page and no comments?

### DISCUSSION THREAD

**Discussion by ymerouani** [kevinzg/facebook-scraper#360] [Q&A]
Hello guys!
Thank you so much for the community and the tool. Its truly a piece of art. I'm a bit of a noob, and after quite a few tries without succeeding I thought it would be best if I contacted you guys.
Im usin the following command in cmd:
`facebook-scraper Visingsö-Alpacka-1164244337013649 -f test4.csv --comments -r --cookies C:\Users\USER\cookies.json`

However, the scraper is only returning the first 6 posts (see image) and no comments from the posts. Same thing happen if I add "-p 50".
![Capture](https://user-images.githubusercontent.com/40887643/122799514-1c95cd80-d2c2-11eb-87ae-3c492159a285.PNG)

Guys, please help. What can I do?

**Comment by neon-ninja** [comment]
This is working fine for me. Do you have a noscript=1 cookie in your cookies.json? If you add the -v flag, do you get any errors in the debug log?

**Comment by ymerouani** [comment]
Hello neon-ninja
Thank you for replying.

I did get an error and was afraid it was my python installation somehow being responsible so I did a fresh install through chocolatey but the error persisted. Here is the command and debug log:

Command: 
`facebook-scraper Visingsö-Alpacka-1164244337013649 -f test1.csv --comments -r --cookies cookies-facebook-com.txt -v`

[facebook_scraper error log.txt](https://github.com/kevinzg/facebook-scraper/files/6692702/facebook_scraper.error.log.txt)

Also, I forgot to add that no the cookies files does not contain anything about "noscript" I even used a different addon to export it from firefox. Still not getting any more posts than these, and also no comments at all. Here is the csv file as well:

[test1_csv.txt](https://github.com/kevinzg/facebook-scraper/files/6692740/test1_csv.txt)

**Reply by neon-ninja** [reply]
That looks like an encoding error - try with the `--encoding utf-8` argument
